setup.py

```diff
@@ -66,6 +67,25 @@
     os.environ["AR"] = "x86_64-linux-gnu-ar"
 
 
+def _get_mac_os_homebrew_prefix() -> str:
```

Comment:
My preference would be a `pathlib.Path` here. I didn't bother because it made the call sites below a bit wordy, and because it doesn't fit with the rest of the project style as far as I saw.